packages/trader/src/Modules/Reports/Containers/statement.jsx

```diff
@@ -198,7 +181,7 @@ class Statement extends React.Component {
                 <ReportsMeta
                     className={is_mx_mlt ? undefined : 'reports__meta--statement'}
                     filter_component={filter_component}
-                    {...this.getReportsMetaProps(is_mx_mlt)}
+                    optional_component={is_mx_mlt && this.getAccountStatistics()}
```

Comment:
Can we move the `is_mx_mlt` check to `client-store`?

Reply:
Sure 👍

Reply:
actually its better to use `standpoint` from client-store instead, we have standpoint.iom, and standpoint.malta